README.md

```diff
@@ -100,7 +98,7 @@ xcode-select --install
 ```
 then:
 ```
-brew install gmp redis graphicsmagick memcached
+brew install gmp graphicsmagick
```

Comment:
@deeptibaghel as part of your check script, on a MAC let's detect if `brew` is available. if not then print a link to `https://brew.sh/` so devs can install it. If brew is available i think we can install these two `gmp graphicsmagick` dependencies for the devs